clouddriver-web/pkg_scripts/postInstall.sh

```diff
@@ -26,6 +28,18 @@ install_kubectl() {
   fi
 }
 
+install_awscli2() {
+  curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64-${AWS_CLI_VERSION}.zip" -o "awscliv2.zip"
```

Comment:
Note arch here is wrong - needs to support arch argument to work on ARM instances.

Reply:
Hrm, that's... potentially problematic.  I need to look at this more then...

Reply:
I can fix it if you want. But I don't think anyone has ever installed the package on an ARM instance. We are probably one of the last companies to actually use these packages, and now we're also migrating to k8s. Looking at the script it is also pretty lacking, without support for anything but AWS. But just let me know if you want me to add support for ARM before we merge this.

Reply:
*wince* I'd love ARM support, but you're probably right on lack of usage. That said I've missed stuff like this before and it's bitten me :(